sktime/forecasting/timesfm_forecaster.py

```diff
@@ -139,15 +139,6 @@ class TimesFMForecaster(_BaseGlobalForecaster):
         # rajatsen91 for google-research/timesfm
         "maintainers": ["geetu040"],
         "python_version": ">=3.10,<3.11",
-        "python_dependencies": [
-            "tensorflow",
-            "einshape",
-            "jax",
-            "praxis",
-            "huggingface-hub",
-            "paxml",
-            "utilsforecast",
-        ],
```

Comment:
I think we should keep these
@fkiraly do you have an opinion here for keeping the dependencies in the tags initially?

Reply:
@Prtm2110 I would suggest we keep these default dependencies